### DISCUSSION TITLE
How is briefcase readthedocs documentation generated?

### DISCUSSION THREAD

**Discussion by wiktor-k** [beeware/briefcase#1471] [Q&A] [answered]
I'd like to extend this site: https://briefcase.readthedocs.io/en/latest/how-to/code-signing/macOS.html

And add a mention on how to get the code signing identity ID (`security find-identity -p basic -v`, [source](https://gist.github.com/txoof/0636835d3cc65245c6288b2374799c43)).

I've browsed through repos in the `beeware` org but I don't see anything regarding to how the documentation is generated.

Thanks for your help! :wave:

**Comment by mhsmith** [comment, marked as answer]
The documentation source files are at https://github.com/beeware/briefcase/tree/main/docs

And if you set up a development environment as shown [here](https://briefcase.readthedocs.io/en/stable/how-to/contribute-code.html), you can build them into HTML by running `tox -e docs`.

**Reply by wiktor-k** [reply]
Oh, great! I think I was grepping for a too long string that was broken between the lines. Thanks a lot! :+1:
